homeassistant/scripts/auth.py

```diff
@@ -45,10 +45,10 @@ def run(args):
     hass.config.config_dir = os.path.join(os.getcwd(), args.config)
     data = hass_auth.Data(hass)
     loop.run_until_complete(data.async_load())
-    loop.run_until_complete(args.func(data, args))
+    loop.run_until_complete(hass, args.func(data, args))
```

Comment:
The command line seems broken. Should this be `loop.run_until_complete(args.func(hass, data, args))`?

Reply:
ah yes.